core/src/parser.rs

```diff
@@ -385,6 +385,7 @@ fn parse_enum_variant(
             fields: fields_named
                 .named
                 .iter()
+                .filter(|f| !is_skipped(&f.attrs))
```

Comment:
Please add an automated test case. Otherwise, this looks good.

Reply:
Added snapshot test.